swan/values.yaml

```diff
@@ -29,6 +29,28 @@ cvmfs:
     - sft-nightlies.cern.ch
   mountOptions:
     hostMountpoint: /var/cvmfs
+  # Prefetcher is provided only by the daemonSet
```

Comment:
can you remove the prefetcher  config down in this file?

Reply:
sorry not this one, there is another 'prefetcher' below for an old development https://github.com/swan-cern/swan-charts/blob/88dc3c42e1221438ac68b82a995f1c38d704ed1d/swan/values.yaml#L235, if you can remove that i don't have to send a commit on top of this pr

Reply:
Fixed with https://github.com/swan-cern/swan-charts/pull/54/commits/88dc3c42e1221438ac68b82a995f1c38d704ed1d